error.c

```diff
@@ -228,16 +228,22 @@ rb_warning_s_aset(VALUE mod, VALUE category, VALUE flag)
 }
 
 /*
- * call-seq:
- *    warn(msg)  -> nil
+ *  call-seq:
+ *     warn(msg)  -> nil
+ *     warn(msg, category: :deprecated)  -> nil
```

Comment:
This call-seq is incorrect, as the category keyword does not default to `:deprecated`.  I would use a single line with `warn(msg, **kw)`, as that is what the implementation accepts, or change the implementation to only allow the `category` keyword and then use `warn(msg, category: nil)`.